README.md

```diff
@@ -1,13 +1,9 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
## Improve the live link of the page on the `README` 
- [ ] Please add a live link to where the page is hosted live. the current link appears to be broken aa it throws a `404` error.